src/backend/parser/gram.y

```diff
@@ -7230,28 +7230,7 @@ DropStmt:	DROP drop_type IF_P EXISTS any_name_list opt_drop_behavior
 					n->concurrent = false;
 					$$ = (Node *)n;
 				}
-			| DROP INDEX CONCURRENTLY any_name_list opt_drop_behavior
-				{
-					DropStmt *n = makeNode(DropStmt);
-					n->removeType = OBJECT_INDEX;
-					n->missing_ok = FALSE;
-					n->objects = $4;
-					n->arguments = NIL;
-					n->behavior = $5;
-					n->concurrent = true;
-					$$ = (Node *)n;
-				}
-			| DROP INDEX CONCURRENTLY IF_P EXISTS any_name_list opt_drop_behavior
-				{
-					DropStmt *n = makeNode(DropStmt);
-					n->removeType = OBJECT_INDEX;
-					n->missing_ok = FALSE;
-					n->objects = $6;
-					n->arguments = NIL;
-					n->behavior = $7;
-					n->concurrent = true;
-					$$ = (Node *)n;
-				}
+			/*GPDB_92_MERGE_FIXME: remove syntax for DROP INDEX CONCURRENTLY*/	
```

Comment:
Maybe just comment out them to benefit future code merging?

So this comment implies we need to do this in the future or this feature is not expected to work on GPDB?

Reply:
Not sure how to track this. GPDB 5 disabled drop index concurrently from syntax level, but 9.2 rewrite the rule and we merged it. So I think we can just remove it. Just a notice for Robert.

Reply:
maybe comment out with a comments is better than remove, so that in future everyone could fix this kind of conflicts easier, especially when someone who are new to merge work on resolving conflicts of related file.

Reply:
make sense